udata_front/theme/gouvfr/__init__.py

```diff
@@ -47,7 +47,23 @@
     nav.Item(_('Reuses'), 'reuses.list'),
     nav.Item(_('Organizations'), 'organizations.list'),
     nav.Item(_('News'), 'posts.list'),
-    nav.Item(_('About'), 'gouvfr.show_page', args={'slug': 'about/ressources'}),
+    nav.Item(_('Starting on data.gouv.fr'), None, items=[
+        nav.Item(
+            _('What is data.gouv.fr?'),
+            'gouvfr.show_page',
+            args={'slug': 'about/ressources'}
+        ),
+        nav.Item(
+            _('How to publish data ?'),
+            'gouvfr.show_page',
+            args={'slug': 'onboarding/producteurs'}
+        ),
+        nav.Item(
+            _('How to use data ?'),
```

Comment:
Maybe this instead:
```suggestion
            _('How to use open data ?'),
```

Reply:
French translations are :
- `Commencer sur data.gouv.fr` for the button
- `Qu'est ce que data.gouv.fr ?`
- `Comment publier des données ?`
- `Comment exploiter des données ?`

Reply:
Yes ok, let's keep `how to use data`!
But for the first, I think we can go for `getting started on data.gouv.fr`. :)

Reply:
Done !